src/DoubleTheDonation/UpdateDonationReceipt.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+
+namespace GiveDoubleTheDonation\DoubleTheDonation;
+
+use Give\Receipt\DonationReceipt;
+use Give\Receipt\Section;
+
+
+class UpdateDonationReceipt {
+
+	/**
+	 * Add employer matching to LEGACY receipt.
+	 *
+	 * @since 1.0.0
+	 */
+	public function renderLegacyRow( $donation ) {
+
+		$instructions = $this->getInstructions( $donation->ID );
+
+
+		if ( isset($instructions['company_name']) ) { ?>
+			<tr>
+				<td scope="row"><strong><?php esc_html_e( 'Employer', 'give-funds' ); ?></strong></td>
+				<td><?php echo $instructions['company_name']; ?></td>
+			</tr>
+
+		<?php }
+
+		if ( isset($instructions['next_steps']) ) { ?>
+			<tr>
+				<td scope="row"><strong><?php esc_html_e( 'Employer Matching', 'give-funds' ); ?></strong></td>
```

Comment:
Why we are using `give-funds` as text domain?

Reply:
Fixed